ports/embed/port/mpconfigport_common.h

```diff
@@ -34,8 +34,13 @@ typedef long mp_off_t;
 
 // Need to provide a declaration/definition of alloca()
 #if defined(__FreeBSD__) || defined(__NetBSD__)
+// BSD
 #include <stdlib.h>
+#elif defined(_WIN32) && (defined(__MINGW64_VERSION_MAJOR) || defined(__MINGW32_VERSION_MAJOR))
+// MinGW-w64; others Windows compilers probably need this too
```

Comment:
Other compilers do need this, e.g. ports/windows/mpconfigport.h also has it for this reason. So I'd leave only the `#elif defined(_WIN32)` bit and mention explicitly it is needed for `alloca()`.

Reply:
Thank you! Good catch. I’ll update this as suggested.